notebooks/nzh-20230521-00-ensembling.ipynb

```diff
@@ -0,0 +1,1004 @@
+{
```

Comment:
<p>I would say keep the original features without multiplying the call model and the features with the multiplication. Then see how logistic regression weights the different features in terms of importance.</p><p> </p><p>Actually reading over the rest of this, if we incorporate the call/nocall information, it should just be the same value but duplicated across all the columns instead of making this new derived column.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dsgt-birdclef/birdclef-2023/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>